docker/2.0/Dockerfile

```diff
@@ -34,7 +34,7 @@ VOLUME /var/lib/mysql
 WORKDIR /mattermost
 
 # Copy over files
-ADD https://github.com/mattermost/platform/releases/download/v2.0.0-rc2/mattermost.tar.gz /
+ADD https://github.com/mattermost/platform/releases/download/v2.0.0/mattermost.tar.gz /
```

Comment:
That's something from master. Its fine to change. Just make sure nothing else not in and let me know.